src/pages/Home/sections/NotableUsersSection.js

```diff
@@ -51,7 +51,7 @@ const NotableUsersSection = () => {
             alignItems='center'
             justify='center'
             style={{ height:'100%' }}>
-            <a href={props.link}>
+            <a href={"/organizations/"+props.alt.replaceAll(" ","")}>
```

Comment:
If this is internal link, we need to use Link component from react router dom

Reply:
updated to use link component instead.